crt_portal/cts_forms/templates/forms/complaint_view/index/complaints_table.html

```diff
@@ -108,23 +116,24 @@
               </td>
             </tr>
             {# this row is hidden by default #}
-            <tr id="tr-additional-{{ datum.report.id }}" hidden>
-              <td colspan="2"></td>
-              <td colspan="4">
-                <div class="td-quickview">
+            <tr class="tr-quickview" id="tr-additional-{{ datum.report.id }}" hidden>
+              {# All screen readers don't yet read the headers attribute correctly, so we've added a redundant aria-label #}
+              {# https://a11ysupport.io/tech/html/headers_attribute #}
+              <td aria-label="Summary" headers="crt-table-summary-header" class="quickview" colspan="6">
+                <label aria-hidden="true" for="tr-quickview-summary-{{ datum.report.id }}" class="td-quickview">
```

Comment:
Because this is inside of a `<td>`, it's redundant for screen readers, as they already read the column header. It'd be like having this for other values:

<img width="143" alt="image" src="https://user-images.githubusercontent.com/15126660/212420308-a11baac0-d192-4df1-a698-95765426e883.png">

Plus, screen readers jump to that header span as a target before jumping to the value, which is inconsistent with how the rest of the table cells work.

That said, the setup is still not ideal - because of the documented lack of support for `headers` in some screen readers, there _is_ an `aria-label` on the td as a backup to avoid it reading:

`Submitted. This is a personal description...`

instead of 

`Personal Description. This is a personal description...`

So our final behavior is:

`(Submitted | Personal Description). Personal Description. This is a personal description...`